eblob_kit.py

```diff
@@ -433,6 +463,31 @@ def file(self):
         """Return file."""
         return self._file
 
+    def close(self):
+        """Close underlying file."""
+        self._file.close()
+
+    @staticmethod
+    def move_back(from_index, to_index):
```

Comment:
Why isn't it just a method of `IndexFile`:
```python
def move(self, path):
    shutil.move(self.path, path)
```

Reply:
It was implemented likewise in first version of this PR, but it seems a violation of class api, as `user` can freely call ```move``` for entity in wrong state, e.g. before any file was closed or written. It could be combined ```close_and_move``` method here, but it looks more rigid then current version.

Reply:
It could be made via close, move, re-open

Reply:
Also it could be made without close and re-open within one filesystem via `os.rename()`.